CONTRIBUTING.md

```diff
@@ -1,33 +1,37 @@
+If you have any questions, or would like to contact team members, you can either
+- Email [Hayden](mailto:me@diatr.us) about your issue/concern (or annoy him on [Twitter](https://twitter.com/Diatrus)), or..
+- Join [Hayden's Discord Server](https://diatr.us/discord), where most team members are capable of providing support and answering questions
 
 ## Submitting Issues
-Any issue with the build system, or any package on the APT repo can be submitted as a Github Issue, and will be tracked as soon as possible.
+Any issue with the build system or a package on the APT repo can be submitted as a Github issue, and will be tracked as soon as possible.
 
-While there isn't an overbaring policy on how to properly submit issues, please be courteous and provide as much information about the situation as possible **in a cohesive matter.** Low effort or low-info issues will be closed.
+While there isn't an overbearing policy on how to properly submit issues, please be courteous and provide as much information about the situation as possible **in a cohesive matter.** Low effort or low info issues will be closed and marked as invalid.
 
 ## Contribution Process
-1. Fork the project, and make changes relevant to what you're trying to add/fix on a seperate branch
-2. Create a Github Pull Request for your change in your specific branch. You should include a description of the change, why you're requesting your change to be made, and where you tested your change
-3. Don't expect your changes to be merge to upstream, since a team member will review your pull request, asking for further possible changes
-4. Once all further changes have been met (if any) and your changes are approved, it will be merged to upstream and the APT repository shortly updated
+1. Fork the project and make changes relevant to what you're trying to add/fix — do this in a seperate branch in your fork
+2. Test your changes made on a **physical** device. If you're adding or updating a package, test your changes with a build of the package you added/updated
+3. Create a Pull Request for your changes in your specific branch — include a description of the change and where you tested your change
+
+Don't expect your changes to be merged and pushed to upstream immediately — your changes will be reviewed by a team member (or Hayden), providing suggested changes which must be met.
+
+Once all further changes have been met (if any) and your changes are approved, they will be merged into upstream. The APT repository will shortly update showcasing your changes.
 
 ### Adding a New Package
-Adding new packages to Procursus is fairly easy and simple. For most packages, you're likely to need 2 files, only: a ``.mk`` file and a ``.control`` file. For documented examples, take a look at the [``grep.mk`` template](./grep.mk.template) and its [control file](./grep.control).
+Adding new packages to Procursus is fairly easy and simple. For most packages, you're likely to need 2 files
```

Comment:
Please mention `./build_tools/new_package.sh`

Reply:
Done in latest commit.